services/movie.py

```diff
@@ -0,0 +1,37 @@
+from typing import Optional
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: Optional[list] = None,
+        actors_ids: Optional[list] = None
+) -> str:
+    movies = Movie.objects.all()
+    if genres_ids:
+        movies = movies.filter(genres__id__in=genres_ids)
+
+    if actors_ids:
+        movies = movies.filter(actors__id__in=actors_ids)
+
+    return movies
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    return Movie.objects.get(id=movie_id)
+
+
+def create_movie(
+        movie_title: str,
+        movie_description: str,
+        genres_ids: Optional[list] = None,
+        actors_ids: Optional[list] = None) -> None:
```

Comment:
add additional typehint for list inner objects like list[YOUR_VALUE_TYPE]
for example: list that contain string elements has typehint list[str]

Reply:
Change Optional[list] to list[int] and delete import in movie.py